src/StreamJsonRpc/MultilingualResources/StreamJsonRpc.pl.xlf

```diff
@@ -151,6 +151,11 @@
           <source>Parameter is not in the form of a single object</source>
           <target state="new">Parameter is not in the form of a single object</target>
         </trans-unit>
+        <trans-unit id="ConflictingMethodNameAttribute" translate="yes" xml:space="preserve">
+          <source>The '{0}' method has multiple MethodNameAttribute defined with different values.</source>
+          <target state="new">The '{0}' method has multiple MethodNameAttribute defined with different values.</target>
```

Comment:
Should "MethodNameAttribute" it be paremeterized instead? Then you can use nameof() or typeof() as format argument.